librz/core/canalysis.c

```diff
@@ -6821,41 +6824,69 @@ RZ_API RZ_OWN RzPVector *rz_core_analysis_bytes(RZ_NONNULL RzCore *core, RZ_NONN
 	int min_op_size = rz_analysis_archinfo(core->analysis, RZ_ANALYSIS_ARCHINFO_MIN_OP_SIZE);
 	min_op_size = min_op_size > 0 ? min_op_size : 1;
 
-	int ret;
-	for (int i = 0, idx = 0; idx < len && (!nops || (nops && i < nops)); i++, idx += ret) {
+	bool bigendian = rz_config_get_b(core->config, "cfg.bigendian");
+	bool asm_sub_var = rz_config_get_i(core->config, "asm.sub.var");
+	core->parser->subrel = rz_config_get_i(core->config, "asm.sub.rel");
+	core->parser->localvar_only = rz_config_get_i(core->config, "asm.sub.varonly");
+
+	const int addrbytes = (int)core->io->addrbytes;
+	const int mask = RZ_ANALYSIS_OP_MASK_ESIL | RZ_ANALYSIS_OP_MASK_IL | RZ_ANALYSIS_OP_MASK_OPEX | RZ_ANALYSIS_OP_MASK_HINT;
+	RzAsmOp asmop;
```

Comment:
This value is never initialized. rz_asm_op_init should be called before being used.

Reply:
https://github.com/rizinorg/rizin/blob/628d04f713bb3fdcd5726f00d4f440ead35dd683/librz/asm/asm.c#L462-L464
but `rz_asm_disassemble` will do this, so is this necessary?